features/conversation-requests-list/use-conversation-requests-list-items.tsx

```diff
@@ -40,8 +40,12 @@ export const useConversationRequestsListItem = () => {
         const isSpamResults = await Promise.allSettled(
           conversations.map(async (conversation) => {
             try {
-              // ! since we check for lastMessage in the select, we can safely assume it's not undefined
-              const lastMessage = conversation.lastMessage!;
+              const lastMessage = conversation.lastMessage;
```

Comment:
I can't think of any consequences of considering an ?empty? conversation in your conversation request being marked as spam? as soon as a message is sent and the app is reloaded the conversation would be refetched anyway and this wouldn't happen, right?